_posts/2021-07-26-release-1.11.4.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: post
+title:  "Apache Flink 1.11.4 Released"
+date:   2021-07-06 00:00:00
+categories: news
+authors:
+- godfrey:
+  name: "Xiaoling He"
+
+---
+
+The Apache Flink community released the next bugfix version of the Apache Flink 1.11 series.
+
+This release includes 56 fixes and minor improvements for Flink 1.11.4. The list below includes a detailed list of all fixes and improvements.
```

Comment:
`1.11.4`-> `1.11.3`

Reply:
The number of issues resolved in 1.11.4 should be 78 according to the [release note](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12349404).